attiny/attiny.ino

```diff
@@ -238,10 +250,18 @@ void receiveEvent(uint8_t howMany) {
       if (howMany != 1) {
         break;
       }
+      gotWDT = true;
       setBlinks(1);
       piWDTCountdown = PI_WDT_RESET_VAL;
       successfulRead = true;
       break;
+    case 0x13:
+      if (howMany != 2) {
+        break;
+      }
+      onWiFi = TinyWireS.receive();
```

Comment:
How about you make `onWiFI` a bool and then handle the read like this:
```c
onWiFi = TinyWireS.receive() == 0x01
```

This makes the code that uses `onWiFi` clearer.

Reply:
Yeah that will look better.